libs/flux/build.go

```diff
@@ -301,6 +301,20 @@ func findModule(mod *modfile.File, logger *zap.Logger) (module.Version, string,
 	// Attempt to find the module in the list of replace values.
 	for _, replace := range mod.Replace {
 		if replace.Old.Path == modulePath {
+			/* If there is a replacement, and the path to the replacement is a relative path,
+			   make the path absolute, relative to the module root.
+			*/
+			if strings.HasPrefix(replace.New.Path, ".") {
```

Comment:
Can you put this logic into a function? Something like:

```
// abspath converts a directory to an absolute path relative
// to the module root.
func abspath(dir string) string {
}
```

Then you can do:

```
replace.New.Path = abspath(replace.New.Path)
```

Also please use `//` style for comments instead of `/* */`. Line comments are considered [preferable](https://golang.org/doc/effective_go.html#commentary) in the effective Go guide.

Reply:
I would rather not put that logic in a function, particularly because it's only used once. I think that we _greatly_ reduce readability when we create functions that are only called once to encapsulate functionality, because then we're pushing onto our mental stacks for very little benefit.

Fixed the comment style.

Reply:
My opinion is when I read this it was difficult for me to necessarily understand what was going on or why. The comment helped, but the two different return paths confused me slightly.

Reply:
Cool. I fixed the different return paths, and I agree that it reads a little better now.